src/legend/svgLegend.ts

```diff
@@ -452,6 +452,9 @@ export class SVGLegend implements ILegend {
             case LineStyle.dotted: {
                 return "2.5,3.1";
             }
+            case LineStyle.dotdash: {
+                return "7,3.1,2.5,3.1";
```

Comment:
it a dashdot style, not a dashdot, correct?

Reply:
Hi @uve - presumably this is to do with the dash preceding the dot in the `stroke-dasharray`? If so, I have revised the order and added a `dashdot` with the aforementioned `stroke-dasharray` so that either is an option. Hopefully this is okay.